src/main/kotlin/com/sourcegraph/cody/edit/sessions/FixupSession.kt

```diff
@@ -94,30 +94,29 @@ abstract class FixupSession(
     CodyAgentService.withAgent(project) { agent ->
       workAroundUninitializedCodebase()
 
+      // All this because we can get the workspace/edit before the request returns!
```

Comment:
I think we really ought to fix this problem too. The client protocol requires you to prepare for receiving information about a session/task that hasn't been created yet, because of a race condition. I've filed a [tracking issue](https://github.com/sourcegraph/jetbrains/issues/1455) for it.